lib/core/test/unit/Cardano/Wallet/Primitive/AddressDerivation/ByronSpec.hs

```diff
@@ -97,11 +98,11 @@ data GenerateKeyFromSeed = GenerateKeyFromSeed
 
 generateTest :: GenerateKeyFromSeed -> Expectation
 generateTest GenerateKeyFromSeed{..} =
-    getKey (generateKeyFromSeed (Passphrase seed) pwd)
+    getKey (generateKeyFromSeed mw pwd)
     `shouldBe`
     getKey rootKey
   where
-    Right (Passphrase seed) = fromMnemonic @'[12] mnem
+    Right mw = fromMnemonic @'[12] mnem
```

Comment:
`either (error . show) id` would provide better failure reasons if this were to ever fail.